server/test/unit/decode_fbs.py

```diff
@@ -1,31 +0,0 @@
-"""
-Code to decode, for testing purposes, the flatbuffer encoded blobs.
-
-This code will need to be updated if fbs/matrix.fbs changes. For more information, see fbs/matrix.fbs and
-server/data_common/fbs/
-"""
-
-import server.data_common.fbs.NetEncoding.Matrix as Matrix
-from server.data_common.fbs.matrix import deserialize_typed_array
```

Comment:
it looks like this function was added to `server/data_common/fbs/fbs_core.py` does it need to be removed from ` server.data_common.fbs.matrix`? or could it be reused?

Reply:
Thanks, good catch.  I had meant to move it, not duplicate it.